packages/zwave-js/src/lib/driver/Driver.ts

```diff
@@ -335,6 +335,8 @@ export interface SendMessageOptions {
 	 * @internal
 	 */
 	tag?: any;
+	/** If a Wake Up On Demand should be requested for the target node. */
+	requestWakeUpOnDemand?: boolean;
```

Comment:
I'm not sure I like the approach of putting this on a message/transaction. I'd add a writable property to `ZWaveNode` (like the `keepAwake` property) and check this when deciding whether to set the flag.

Reply:
See my top level [comment](https://github.com/zwave-js/node-zwave-js/pull/3332#issuecomment-921349465) where I explain why I went away from having a manual flag.